src/components/Graph.react.js

```diff
@@ -160,6 +160,14 @@ class PlotlyGraph extends Component {
                 }
             }
         });
+        gd.on('plotly_restyle', eventData => {
+            const restyleData = filterEventData(gd, eventData, 'restyle');
+            if (!isNil(restyleData)) {
+                if (setProps) {
```

Comment:
:hocho: Think we can remove these checks now ?

Reply:
good call. In case @Marc-Andre-Rivet is already working on removing the others I'll just do this one for now.

Reply:
🔪 in https://github.com/plotly/dash-core-components/pull/483/commits/8e7806b5135c653b6f3881a72487fc387fb8d3c6 - but as commented I only changed the new prop for now.